src/ripple/overlay/impl/OverlayImpl.cpp

```diff
@@ -932,7 +1005,25 @@ OverlayImpl::processRequest (http_request_type const& req,
     msg.insert("Server", BuildInfo::getFullVersionString());
     msg.insert("Content-Type", "application/json");
     msg.insert("Connection", "close");
-    msg.body()["overlay"] = crawl();
+    msg.body()["version"] = Json::Value(1u);
```

Comment:
I just stumbled upon this `"version": 1` field in the `/crawl` json data. What does it signify and under which circumstances would this get changed/incremented?

Reply:
It would be used for breaking changes, i.e. a field is removed or if the representation of the data changed significantly.